app/mailers/interest_subscription_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class InterestSubscriptionMailer < ApplicationMailer
+  def daily_album_mailer
+    @user = params[:user]
+    @album = params[:album]
```

Comment:
in all workers and mailers better to path just ids of any entity in params and then retrieve them from db, instead of passing entire object